bigtop_toolchain/manifests/python.pp

```diff
@@ -40,20 +40,24 @@
     ensure => present
   }
 
-  if ($architecture in ['aarch64']) {
-    case $operatingsystem{
-      /(?i:(fedora|ubuntu|debian))/: {
-        package { 'python2' :
-          ensure => present
-        }
-      }
-      /(?i:(centos|redhat|rocky))/: {
-        if (versioncmp($operatingsystemmajrelease, '8') == 0) {
-          package { 'python2' :
-            ensure => present
-          }
-        }
-      }
+  if ($architecture in ['aarch64', 'ppc64le']) {
+    exec { "download_python2.7":
+      cwd => "/usr/src",
+      command => "/usr/bin/wget https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tgz --no-check-certificate && /usr/bin/mkdir Python-2.7.18 && /bin/tar -xvzf Python-2.7.18.tgz -C Python-2.7.18 --strip-components=1 && cd Python-2.7.18",
```

Comment:
For what reason is `--no-check-certificate` required here?

Reply:
While I don't know the intention of the original author, it seems to work without the option. I removed it.